### DISCUSSION TITLE
Generate thumbnail  for video

### DISCUSSION THREAD

**Discussion by saitauthao** [diced/zipline#376] [Ideas]
would like a generated thumbnail for uploaded video when looking at it from the gallery.

**Comment by fletchowns** [comment]
How about embedding the video itself and play on mouseover, pause on mouse out? Like this:

https://user-images.githubusercontent.com/153107/237033611-03f68358-aca7-4ba3-8e3b-77b10fcdef4a.mp4

**Reply by saitauthao** [reply]
> How about embedding the video itself and play on mouseover, pause on mouse out? Like this:
> 
>  firefox_UQeGijvWRJ.mp4

Yes, exactly what I was looking for.

**Comment by diced** [comment]
This is now added in https://github.com/diced/zipline/pull/413, and was merged (https://github.com/diced/zipline/commit/5c424a2c6d7e473720499bafb9d6e1cef5ecd299) today.

It generates thumbnails for videos on an interval, every 10 minutes by default - though it can be configured like every other interval. Once a video has a thumbnail it will show the thumbnail with a play button over it.
